nexus/src/app/sagas/region_replacement_start.rs

```diff
@@ -221,6 +221,13 @@ async fn srrs_get_existing_datasets_and_regions(
         .await
         .map_err(ActionError::action_failed)?;
 
+    // XXX for now, bail out if requesting the replacement of a read-only region
```

Comment:
Should this be an issue link?

Reply:
You're right, opened https://github.com/oxidecomputer/omicron/issues/6172